index.html

```diff
@@ -74,6 +74,176 @@ <h2>Let's connect</h2>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="skills-ul">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <a class ="link-btn" href="https://cindykandie.xyz">
+                    See Project
+                </a>
+            </div>
+        </article><article class="project-card project_one">
+            <img src="assets/tonic2.svg" alt="">
+            <div class="project_info">
+                <h1>Tonic</h1>
+                <ul>
+                    <li>Canopy</li>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="skills-ul">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <a class ="link-btn" href="https://cindykandie.xyz">
+                    See Project
+                </a>
+            </div>
+        </article><article class="project-card project_one">
+            <img src="assets/multi2.svg" alt="">
+            <div class="project_info">
+                <h1>Multi-Post
+                    Stories</h1>
+                <ul>
+                    <li>Canopy</li>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="skills-ul">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <a class ="link-btn" href="#">
+                    See Project
+                </a>
+            </div>
+        </article>
+    </section>
+
+    <section class="about-section">
+        <article>
+            <h1>About <br>
+                Myself
+            </h1>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+        </article>
+        <div class="contact-me">
+            <h2>Let's connect</h2>
+            <ul class="socials">
+                <li>
+                    <a href="https://twitter.com/cindykandie">
+                        <img src="assets/twitter.svg" alt="My twitter page" >
+                    </a>
+                </li>
+               
+                <li>
```

Comment:
- [ ] Impressive job @cindykandie, I just have one major suggestion for your `about me` section. Here on a `375px` screen size,  your social media icon seems to overlap with the `see project` button. Lets fix this to be even more exact.
### Figma Design 
![button2](https://user-images.githubusercontent.com/69077061/205425315-dedded33-b17f-468a-b76c-85600404382e.PNG)
### Your design 
![button](https://user-images.githubusercontent.com/69077061/205425359-7df938d9-bc4d-4c72-be56-ee38d229c401.PNG)